cmd/queue/main.go

```diff
@@ -66,10 +67,10 @@ const (
 	drainSleepDuration = 30 * time.Second
 
 	// certPath is the path for the server certificate mounted by queue-proxy.
-	certPath = queue.CertDirectory + "/tls.crt"
+	certPath = queue.CertDirectory + "/" + certificates.SecretCertKey
```

Comment:
I guess the next step is to use the mounted secret to enable tls at the queue proxy internal servers.

Reply:
The queue-proxy already mounts the secret when enabling the option.

https://github.com/knative/serving/pull/13005/files#diff-fac493c4e5816a0735e649ef20f47c101d4c5dab982c434d014708df695970caR142-R144